api/docs.go

```diff
@@ -254,6 +254,18 @@ const docTemplate = `{
                         "description": "Comma separated list of statuses to optionally filter on",
                         "name": "status",
                         "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "description": "Comma separated list of origins to filter (red_hat,external)",
```

Comment:
```suggestion
                        "description": "Comma separated list of origins to filter on (red_hat,external)",
```
All other descriptions use the word "on" and it would be nice to be consistent